### PR TITLE
workload: fix quantile values in openmetrics exporter

### PR DESCRIPTION
**Pull request by sambhav-jain-16**
`hdrHistogram` library has quantiles in percentages rather than absolute
values. Openmetrics standard forces the quantiles values in `summary`
metric to be absolute values b/w 0 and 1 inclusive.
https://github.com/prometheus/OpenMetrics/blob/296468bc2359ebac83f24301b54a0871f2268016/specification/OpenMetrics.md?plain=1#L294

Earlier the exporter would export the percentage value, this change
intends to make it absolute to adhere to openmetrics standard

Epic: none

Release note: None

### CODE REVIEW

**Review by herkolategan — COMMENTED** [review]
Which other tests use `50.0` vs. `0.5`? Curious as the OpenMetrics standard seems to suggest this format: https://github.com/prometheus/OpenMetrics/blob/main/specification/OpenMetrics.md#summary-1

**Comment by sambhav-jain-16** [comment]
> Which other tests use `50.0` vs. `0.5`? Curious as the OpenMetrics standard seems to suggest this format: https://github.com/prometheus/OpenMetrics/blob/main/specification/OpenMetrics.md#summary-1

Every other tests using the `workload` binary and exporting the histogram summaries from it eg tpcc, ycsb etc. 
Actually I checked and looks like this needs to be fixed in the openmetrics exporter itself rather than here.

I'm using `expfmt.MetricFamilyToOpenMetrics` from prometheus library so I assumed it will emit in right format. Looks like I was wrong.
I'll push the fixes in the workload binary in this PR itself. Thanks @herkolategan for pointing out.

File: pkg/workload/histogram/exporter/exporter_test.go [hunk 1]
```diff
@@ -209,7 +209,7 @@ func TestOpenMetricsFileWithJson(t *testing.T) {
 							return errors.Errorf("invalid summary count: %f", countValue)
 						}
 					}
-
+					quantile *= 100
```
Comment (nameisbhaskar) [inline pkg/workload/histogram/exporter/exporter_test.go:212]: Can you please add a comment here as well mentioning why we have done *100?

**Review by nameisbhaskar — APPROVED** [review]
LGTM! Just one minor comment.

File: pkg/workload/histogram/exporter/exporter_test.go (hunk 1 above)
Comment (sambhav-jain-16) [reply pkg/workload/histogram/exporter/exporter_test.go:215]: Done

**Comment by sambhav-jain-16** [comment]
TFTR! 

bors r=@nameisbhaskar

**Comment by sambhav-jain-16** [comment]
Build passed: https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Roachtest_Openmetrics_Gce/17834130?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true&expandBuildTestsSection=true

**Pull request merged**